Backend/package.json

```diff
@@ -8,7 +8,7 @@
     "prebuild": "npm run clean:build",
     "build": "npm run prisma:generate && tsc --build --verbose tsconfig.build.json && tsc-alias -p tsconfig.json",
     "start": "FORCE_COLOR=2 node ./dist/index.js",
-    "dev": "npm run prisma:generate && NODE_ENV=development ts-node-dev --project tsconfig.dev.json -r tsconfig-paths/register --transpile-only --respawn --exit-child src/index.ts",
```

Comment:
Why remove Node_Env?

Reply:
The same reason for modifying the .env file suffix, as the script did not work without these modifications. You can try it yourself.

Reply:
What is the error you are getting of using that command as it is?

Reply:
This error: 'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

Reply:
because it is compatible with linux not windows. windows has slight different syntax.
Anyway, you can ignore it for now.